src/reporter.js

```diff
@@ -29,12 +32,105 @@ class Reporter {
             accessKey: this.accessKey,
             headers: {'User-Agent': `cypress-reporter/${reporterVersion}`}
```

Comment:
Totally unrelated to your PR, but good opportunity to fix 😅 
```suggestion
            headers: {'User-Agent': `testcafe-reporter/${reporterVersion}`}
```